cellprofiler/modules/plugins/measurementtemplate.py

```diff
@@ -257,39 +358,38 @@ def run(self, workspace):
         #
         for n, m in self.get_zernike_indexes():
             # Compute the zernikes for each object, returned in an array
-            zr, zi = self.measure_zernike(
-                    pixels, labels, indexes, centers, radius, n, m)
+            zr, zi = self.measure_zernike(pixels, labels, indexes, centers, radius, n, m)
+
             # Get the name of the measurement feature for this zernike
             feature = self.get_measurement_name(n, m)
+
             # Add a measurement for this kind of object
             if m != 0:
-                meas.add_measurement(input_object_name, feature, zr)
-                #
+                measurements.add_measurement(input_object_name, feature, zr)
+
                 # Do the same with -m
-                #
                 feature = self.get_measurement_name(n, -m)
-                meas.add_measurement(input_object_name, feature, zi)
+                measurements.add_measurement(input_object_name, feature, zi)
             else:
                 # For zero, the total is the sum of real and imaginary parts
-                meas.add_measurement(input_object_name, feature, zr + zi)
-            #
+                measurements.add_measurement(input_object_name, feature, zr + zi)
+
             # Record the statistics.
-            #
-            zmean = np.mean(zr)
-            zmedian = np.median(zr)
-            zsd = np.std(zr)
+            zmean = numpy.mean(zr)
+            zmedian = numpy.median(zr)
+            zsd = numpy.std(zr)
             statistics.append([feature, zmean, zmedian, zsd])
 
     ################################
     #
-    # DISPLAY
+    # "display" lets you use matplotlib to display your results.
     #
-    def display(self, workspace, figure=None):
+    ################################
+    def display(self, workspace, figure):
```

Comment:
It seems like this has to be overridden for all measurement modules. Does it make sense to standardize this processing/displaying like it's been done in the `ImageProcessing` and `ObjectProcessing` modules, or are the implementations too specific?

Reply:
Great question. A quick look shows me that most measurement modules only display `workspace.statistics` (there are a few exceptions). That could be made the default behavior in a measurement module superclass.